Src/Idioms/ConstructorInitializedMemberAssertion.cs

```diff
@@ -250,11 +260,53 @@ public override void Verify(FieldInfo fieldInfo)
             }
         }
 
+        private static bool CouldLeadToFalsePositive(Type type)
+        {
+            if (type.IsEnum)
+            {
+                //Check for a default value only enum
+                var values = Enum.GetValues(type);
+                return values.Length == 1 && (int)values.GetValue(0) == default(int);
+            }
+
+            return false;
+        }
+
+        private static ConstructorInitializedMemberException BuildExceptionDueToPotentialFalsePositive(MemberInfo propertyOrField)
+        {
+            var message = string.Format(CultureInfo.CurrentCulture, "Unable to properly detect a successful intialization due to {0} of type enum with a single default value.{3}Declaring type: {1}{3}Reflected type: {2}{3}", propertyOrField.Name, propertyOrField.DeclaringType.AssemblyQualifiedName, propertyOrField.ReflectedType.AssemblyQualifiedName, Environment.NewLine);
+
+            if (propertyOrField is FieldInfo)
+            {
+                return new ConstructorInitializedMemberException(propertyOrField as FieldInfo, message);
+            }
+            else if (propertyOrField is PropertyInfo)
+            {
+                throw new ConstructorInitializedMemberException(propertyOrField as PropertyInfo, message);
+            }
+            else
+            {
+                throw new ArgumentException("Must be a property or field", "propertyOrField");
+            }
+        }
+
         private ExpectedAndActual BuildSpecimenFromConstructor(
             ConstructorInfo ci, MemberInfo propertyOrField)
         {
             var parametersAndValues = ci.GetParameters()
-                .Select(pi => new {Parameter = pi, Value = this.builder.CreateAnonymous(pi)})
+                .Select(pi =>
+                {
+                    var value = this.builder.CreateAnonymous(pi);
+                    
+                    //ensure enum isn't getting the default value, otherwise we won't be able to determine whether initialization occurred
+                    if (pi.ParameterType.IsEnum && (int)value == default(int)) value = this.builder.CreateAnonymous(pi);
```

Comment:
I'd like to discuss this.

I was a bit uncomfortable doing this here thinking there might be a more declarative approach.  Perhapsa customization which overrides `EnumGenerator`'s behavior - say `EnumGeneratorNoDefault`.  Trouble is I wasn't comfortable moving this towards 'IFixture'.  Maybe there's a better way?